src/test/java/com/debatetimer/service/parliamentary/ParliamentaryServiceTest.java

```diff
@@ -0,0 +1,143 @@
+package com.debatetimer.service.parliamentary;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.junit.jupiter.api.Assertions.assertAll;
+
+import com.debatetimer.BaseServiceTest;
+import com.debatetimer.controller.exception.custom.DTClientErrorException;
+import com.debatetimer.domain.member.Member;
+import com.debatetimer.domain.parliamentary.ParliamentaryTable;
+import com.debatetimer.domain.parliamentary.ParliamentaryTimeBox;
+import com.debatetimer.dto.parliamentary.request.ParliamentaryTableCreateRequest;
+import com.debatetimer.dto.parliamentary.request.TableInfoCreateRequest;
+import com.debatetimer.dto.parliamentary.request.TimeBoxCreateRequest;
+import com.debatetimer.dto.parliamentary.response.ParliamentaryTableResponse;
+import java.util.List;
+import java.util.Optional;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+
+class ParliamentaryServiceTest extends BaseServiceTest {
+
+    @Autowired
+    ParliamentaryService parliamentaryService;
+
+    @Nested
+    class save {
+
+        @Test
+        void 의회식_토론_테이블을_생성_한다() {
```

Comment:
[사소한 의견] ParliamentaryServiceTest의 테스트 메서드가 서술어 어간이랑 어미 사이에 띄어쓰기 되어있어요. 붙이는 건 어떨까요?

```suggestion
        @Test
        void 의회식_토론_테이블을_생성한다() {
```

Reply:
✅ 모두 반영해주었습니다!